docs/user/interop.rst

```diff
@@ -325,17 +325,13 @@ pointers and do not have a corresponding first-class values backing them.
   a type-level natural number. Natural numbers are types that are composed of
   base naturals ``Nat._0, ... Nat._9`` and an additional ``Nat.DigitN``
   constructors, where ``N`` refers to number of digits in the given number. 
-  So for example number ``1024`` is going to be encoded as following:
-
-  .. code-block:: scala
+  So for example number ``1024`` is going to be encoded as following::
```

Comment:
I'm a bit confused by this change. This is probably my lack of knowledge about the .rst format. Would you mind explaining what is going here, or point me to relevant documentation?

Reply:
Thank you for the timely review.  Yes, there is a bit of wizardry going on there. I could find no good way
to document it inline (but may have just now. I will try that after this)

The primary documentation is (sphinx)[ https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=code-block#showing-code-examples ]. Follow the 
(Pygments)[https://pygments.org/docs/lexers/] link to show that a scala parser is available.

I found the (tutorial) [https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#inserting-code-and-literal-blocks]
more useful.

Basically, there is no visible change in the output.  Prior to this PR, Sphinx warned that it could not 
parse those two blocks as Scala, so it was not doing any Scala highlighting for them.   I took Sphinx
at its word and dropped the Scala specification.   I did not try to figure out why Sphinx could not parse
the contained code because I could not economically change Sphinx.    I suspect that it has something
to do with the leading underbars, especially when the rest looks like a number.

Turns out that Sphinx can parse & highlight the first changed block (as the default python).
 I believe the highlighting is proper but can check again. So that is actually a step to make that block
 consistent with the rest of the file.

The second block did not have highlighting before this PR and does not after it.  

I have been told that if one leads a good life, one never gets this deep into Rich Structured Text details.

Does this address your concerns?   I'll see about adding a comment so that it is not visible in the end
product but briefly explains the wizardry in the file: reduced astonishment.

Reply:
While chasing down the documentation to explain this change, I discovered a way to enter comments
in the source file.  I believe it is far better to not have surprises in the file but where they are unavoidable
a brief comment helps the next person understand.